### DISCUSSION TITLE
how to delete my project on the cloud?

### DISCUSSION THREAD

**Discussion by SerhiyKSV** [opengisch/QField#2464] [Q&A]
I have created several test projects that I do not use. How can I delete a project on the cloud that I don't need?

**Comment by suricactus** [comment]
Hi @SerhiyKSV . Thanks for trying out QFieldCloud. 

You can delete your project from the Web UI if you follow:
1) you click on your project to be deleted
2) go to "Settings" menu on the left
3) in the bottom you will see in red "Delete this project" section
4) make sure you are on the right project, this action is irreversible!
5) click on the red button "Delete project". You will see a new dialog.
6) make sure again this is the correct project. If yes, then type the required text and press "Delete project".
7) Done! Hope it was the correct project.

![image](https://user-images.githubusercontent.com/2820439/145635806-d8b1d0c1-daca-47a6-8f8b-d1801fe05d77.png)

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).
